docs/src/components/action/LogoWithCopyMenu.tsx

```diff
@@ -66,7 +66,7 @@ export default function LogoWithCopyMenu({
         aria-label="Go to homepage"
         onContextMenu={handleContextMenu}
         sx={{
-          cursor: 'default',
+          cursor: 'pointer',
```

Comment:
```suggestion
```

Reply:
Hey @oliviertassinari 
Thank you for your feedback! I see your point regarding cursor: 'default'. Would you like me to make further adjustments to remove it entirely or modify other related styles? Please let me know how you'd like me to proceed, and I’ll be happy to make the necessary changes.